rfd/0013-automating-bench-soak-tests.md

```diff
@@ -0,0 +1,61 @@
+# RFD 13 - Automating `tsh bench` soak tests and data collection
+## Purpose
+Automate the running of tests and collection of data (profiles and dashboards) for the soak test portion of the test plan.
+
+### Overview
+At the moment, `tsh bench` tool is used to manually perform and collect data for soak tests following commands against a node.
+
+Soak Tests    
+Run 4hour soak test with a mix of interactive/non-interactive sessions:   
+`tsh bench --duration=4h user@teleport-monster-6757d7b487-x226b ls`   
+`tsh bench -i --duration=4h user@teleport-monster-6757d7b487-x226b ps uax`
+
+## Technical Details
+### Cluster Setup
+Cluster setup and load tests will continue to be a manual process at the moment and is out of scope for this RFD.
```

Comment:
@klizhentas Do you want to keep running this test against a single node cluster or on a 10k cluster? I know there were some discussions around the 10k cluster test being more realistic.

Reply:
Running bench against proxy or auth with 10k nodes would require we automate the scaling. That is possible, but a bit much for this RFD. Of course, let's discuss it though